src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy

```diff
@@ -70,6 +70,7 @@ class DependencyManager {
             this.sampleManagementDataSource = new OpenbisDataSource(configManager, userID)
         } catch (Exception e) {
             log.error("Error when trying to connect to openBIS.", e)
+            throw e
         }
         
```

Comment:
I think this try catch block can be completely removed. The exceptions are captured by the [calling class](https://github.com/qbicsoftware/sample-tracking-update-portlet/blob/602f1be5ac24ff522b1f2899dd29b47ea9d1ccc9/src/main/groovy/life/qbic/portal/sampletracking/SampleUpdatePortlet.groovy#L22) already and logged there as well. From the StackTrace we can easily see, which method invocation failed.

Reply:
Yes, you could do that but it would be helpful to find the log entry. So maybe change to 
```suggestion
        } catch (Exception e) {
            log.error("Error when trying to connect to openBIS.")
            throw e
        }
        
```
?